api/src/serge/routers/model.py

```diff
@@ -18,140 +18,140 @@
         "gpt4-alpaca-lora-30b.ggmlv3.q5_1.bin",
         24.4e9,
     ],
-    "AlpacaLoRA-65B": [
+    "AlpacaLoRA-65B-q5_K_M": [
         "TheBloke/alpaca-lora-65B-GGML",
-        "alpaca-lora-65B.ggmlv3.q5_1.bin",
-        48.97e9,
+        "alpaca-lora-65B.ggmlv3.q5_K_M.bin",
+        46.2e9,
     ],
-    "OpenAssistant-30B": [
+    "OpenAssistant-30B-q6_K": [
         "TheBloke/OpenAssistant-SFT-7-Llama-30B-GGML",
-        "OpenAssistant-SFT-7-Llama-30B.ggmlv3.q5_1.bin",
-        24.4e9,
+        "OpenAssistant-SFT-7-Llama-30B.ggmlv3.q6_K.bin",
+        26.7e9,
     ],
-    "GPT4All-13B": [
+    "GPT4All-13B-q6_K": [
         "TheBloke/GPT4All-13B-snoozy-GGML",
-        "GPT4All-13B-snoozy.ggmlv3.q5_1.bin",
-        9.76e9,
+        "GPT4All-13B-snoozy.ggmlv3.q6_K.bin",
+        10.7e9,
     ],
-    "StableVicuna-13B": [
+    "StableVicuna-13B-q6_K": [
         "TheBloke/stable-vicuna-13B-GGML",
-        "stable-vicuna-13B.ggmlv3.q5_1.bin",
-        9.76e9,
+        "stable-vicuna-13B.ggmlv3.q6_K.bin",
+        10.7e9,
     ],
-    "Vicuna-v1.1-7B": [
+    "Vicuna-v1.1-7B-q6_K": [
```

Comment:
For all the 7B models, can you also leave the old ones. That way we dont force everyone to use +8GB of ram